### DISCUSSION TITLE
Library module: select/delete multiple collections

### DISCUSSION THREAD

**Discussion by pedrorrodriguez** [aurelienpierreeng/ansel#791] [Ideas]
Currently you can only select/delete one at a time. It'd be great to be able to do it with several in one go.

**Comment by aurelienpierre** [comment]
Fixed with https://github.com/aurelienpierreeng/ansel/commit/b9f5b4f1e67d5be20e07d4bd04409e0041da702d

**Comment by sahlirc** [comment]
While the commit fixed the possibility to select multiple folders, and allowed actions triggered upon mouse right clicking, it would be great to include the display of all images from the selected folders in the lighttable to facilitate culling and/or presentation.
Presently, images of only the first selected folder are displayed in the lighttable, not those of all selected folders, yet Ansel knows their path since it can execute the actions mentioned above. I have no idea whether this is a technical limitation. Right now if I need to display all images from selected folders, I must select them by color coding or star ratings or through SQL queries....
